connectors/mongo/connector.go

```diff
@@ -377,6 +392,12 @@ func (mc *Connector) StartReadToChannel(flowId iface.FlowID, options iface.Conne
 					col := task.Def.Col
 					collection := mc.Client.Database(db).Collection(col)
 					cursor, err := collection.Find(mc.FlowCtx, bson.D{})
+
+					//retrieve namespace status struct for this namespace to update accordingly
+					ns := iface.Namespace{Db: db, Col: col}
+					nsStatus := mc.Status.ProgressMetrics.NamespaceProgress[ns]
+					mc.ProgressTracker.TaskStartedProgressUpdate(nsStatus, task.Id)
```

Comment:
it looks like nsStatus can be calculated in the progress tracker- seems like all we need to pass in in ns.